server/src/main/java/org/elasticsearch/common/network/HandlingTimeTracker.java

```diff
@@ -9,58 +9,20 @@
 
 package org.elasticsearch.common.network;
 
-import java.util.concurrent.atomic.LongAdder;
+import org.elasticsearch.common.metrics.ExponentialBucketHistogram;
 
 /**
  * Tracks how long message handling takes on a transport thread as a histogram with fixed buckets.
  */
-public class HandlingTimeTracker {
+public class HandlingTimeTracker extends ExponentialBucketHistogram {
```

Comment:
I decided to keep the `HandlingTimeTracker` because the decision to have 17 + 1 buckets is contextual and should be fixed because it's used in serialization.

`HandlingTimeTracker` is now just an `ExponentialBucketHistogram` of a specific size.